pkgs/top-level/python-packages.nix

```diff
@@ -8304,14 +8304,14 @@ in {
 
   django_tagging = callPackage ../development/python-modules/django_tagging { };
 
-  django_tagging_0_3 = self.django_tagging.overrideAttrs (attrs: rec {
-    name = "django-tagging-0.3.6";
+  django_tagging_0_4_3 = self.django_tagging.overrideAttrs (attrs: rec {
+    name = "django-tagging-0.4.3";
 
     src = pkgs.fetchurl {
       url = "mirror://pypi/d/django-tagging/${name}.tar.gz";
-      sha256 = "03zlbq13rydfh28wh0jk3x3cjk9x6jjmqnx1i3ngjmfwbxf8x6j1";
+      sha256 = "0617azpmp6jpg3d88v2ir97qrc9aqcs2s9gyvv9bgf2cp55khxhs";
     };
-    propagatedBuildInputs = with self; [ django ];
+    propagatedBuildInputs = with self; [ django_1_8 ];
```

Comment:
@basvandijk this is incorrect; we don't depend on specific versions. Instead, when you need this package with this version of Django the package set needs to be overridden. Can you open a PR with a fix?

Reply:
What if I then do `python.withPackages(ps: with ps; [ graphite_web django_tagging ])`. Then I'll get two versions of `django_tagging` in a closure. We can't have that!

The only solution to picking the correct versions is overriding the package set, e.g. `python.override { packageOverrides = super: self: { }; }`

Reply:
@FRidh good point. Will you be at NixCon this weekend? If so, it would be cool if you could explain to me how to accomplish this.